lib/adb.js

```diff
@@ -24,7 +24,8 @@ const DEFAULTS_OPTS = {
   javaVersion: null,
   suppressKillServer: null,
   jars: {},
-  helperJarPath: JAR_PATH
+  helperJarPath: JAR_PATH,
+  adbPort: 5037
```

Comment:
I'd prefer `5037` to be pulled out as a `const`, so it's obvious what the constants are.